website/versioned_docs/version-21.5.2/faq.md

```diff
@@ -32,7 +32,8 @@ non-standard DevTools Protocol used by Chrome).
 The goals of the project are:
 
 - Provide a slim, canonical library that highlights the capabilities of the
-  [DevTools Protocol](https://chromedevtools.github.io/devtools-protocol/).
+  [DevTools Protocol](https://chromedevtools.github.io/devtools-protocol/) and
+  [WebDriver BiDi](https://w3c.github.io/webdriver-bidi/).
```

Comment:
Maybe add a link to "is Puppeteer BiDi ready"?

Reply:
Or add another question "Q: Does Puppeteer support WebDriver BiDi?"

Reply:
Added a new question